capabilities/host.go

```diff
@@ -190,3 +275,44 @@ func (h *Host) GetResource(req GetResourceRequest) ([]byte, error) {
 
 	return responsePayload, nil
 }
+
+// Verify_cert verifies cert's trust against the passed cert_chain, and
+// expiration and validation time of the certificate.
+// Accepts 3 arguments:
+//   - cert: PEM-encoded certificate to verify.
+//   - cert_chain: list of PEM-encoded certs, ordered by trust usage
+//     (intermediates first, root last). If empty, certificate is assumed trusted.
+//   - not_after: string in RFC 3339 time format, to check expiration against.
+//     If None, certificate is assumed never expired.
+func (h *Host) VerifyCert(cert Certificate, certChain []Certificate, notAfter string) (bool, error) {
```

Comment:
This comment refers to the whole `VerifyCert` function. The function is not building the right payload.
The CertificationVerificationRequest should be define in this way:

```go
type CertificateVerificationRequest struct {
   certificate string
   certificateChain []string
   notAfter string
}
```

See the protocol definition [here](https://docs.kubewarden.io/writing-policies/spec/host-capabilities/crypto).

Because of that, the signature of this function can be changed to be:

```go
func (h *Host) VerifyCert(cert string, certChain []string, notAfter string) (bool, error)
```

That's because this function expects all the certificates to be PEM encoded. There's no need to use the `Certificate` type